src/picomsg/src/oper_db_writer.cpp

```diff
@@ -59,3 +59,50 @@ bool oper_db_writer_terminator_state_change(const uint32_t terminator_id, const
 
 	return true;
 }
+
+bool oper_db_writer_set_terminator_tunnel_counters(const uint32_t terminator_id, const uint64_t count) {
+	auto oper_db_client = picomsg_get_oper_db_client();
+	std::string terminator_id_str = std::to_string(terminator_id);
+	const char* keys[] = { terminator_id_str.c_str()};
+	auto db_obj = oper_db_client_create_db_object(*oper_db_client, LIBOPER_DB_CLIENT_CLASS_ID_TERMINATOR, keys);
+	if (! db_obj) {
+		PICOMSG_LOG(PICOMSG_LOG_ERR, "failed to create oper DB object for class ID LIBOPER_DB_CLIENT_CLASS_ID_TERMINATOR");
+		return false;
+	}
+	
+	if (! db_obj->u64_set(db_obj.get(), LIBOPER_DB_CLIENT_FIELD_ID_TERMINATOR_TUNNEL_COUNTER, count)) {
+		PICOMSG_LOG(PICOMSG_LOG_ERR, "failed to set oper DB object LIBOPER_DB_CLIENT_FIELD_ID_TERMINATOR_TUNNEL_COUNTER to %lu", count);
+		return false;
+	}
+
+	if (! (*oper_db_client)->set((*oper_db_client).get(), db_obj.release(), 0)) {
+		PICOMSG_LOG(PICOMSG_LOG_ERR, "failed to write terminator[%u] tunnel counter to oper DB", terminator_id);
+		return false;
+	}
+
+	return true;
+}
+
+bool oper_db_writer_set_term_vrf_tunnel_counter(const uint32_t terminator_id, const char *tenant_id, 
+                                                const char *account_id, const char *vrf_name, const uint64_t count) {
+	auto oper_db_client = picomsg_get_oper_db_client();
+	std::string terminator_id_str = std::to_string(terminator_id);
+	const char* keys[] = { terminator_id_str.c_str(), tenant_id, account_id, vrf_name };
+	auto db_obj = oper_db_client_create_db_object(*oper_db_client, LIBOPER_DB_CLIENT_VRF_TUNNEL_COUNT_ID_TERMINATOR, keys);
+	if (! db_obj) {
+		PICOMSG_LOG(PICOMSG_LOG_ERR, "failed to create oper DB object for class ID LIBOPER_DB_CLIENT_VRF_TUNNEL_COUNT_ID_TERMINATOR");
+		return false;
+	}
+
+	if (! db_obj->u64_set(db_obj.get(), LIBOPER_DB_CLIENT_FIELD_ID_VRF_TERMINATOR_TUNNEL_COUNTER, count)) {
+		PICOMSG_LOG(PICOMSG_LOG_ERR, "failed to set oper DB object  LIBOPER_DB_CLIENT_FIELD_ID_TERMINATOR_TUNNEL_COUNTER to %lu", count);
```

Comment:
Remove the duplicated space between `object` and `LIBOPER_DB_CLIENT_FIELD_ID_TERMINATOR_TUNNEL_COUNTER`